docs/cookbook/shop/disabling-guest-checkout.rst

```diff
@@ -19,6 +19,44 @@ That will do the trick. Now, when a guest user tries to click the checkout butto
 they will be redirected to the login/registration page, where after they sign in/sign up they
 will be redirected to the checkout addressing step.
 
+How to disable guest checkout in new API?
+=========================================
+
+Our new API let's any anonymous user to proceed through the checkout out of the box (of course if API is enabled).
+In order to disable checking out for anonymous user you have to change the same file but use different routes.
+Here we will let user to only:
+
+    * Get new cart
+    * Get the order by token value
+    * Let the user to add an item to the cart
+
+We will modify the same file as in example on top, the ``security.yaml``:
+
+.. code-block:: yaml
+
+    # config/packages/security.yaml
+    security:
+        access_control:
+    -       - { path: "%sylius.security.new_api_shop_regex%/.*", role: IS_AUTHENTICATED_ANONYMOUSLY }
+    +       - { path: "%sylius.security.new_api_shop_regex%/orders", method: POST, GET , role: IS_AUTHENTICATED_ANONYMOUSLY }
+    +       - { path: "%sylius.security.new_api_shop_regex%/orders/.*/items", method: POST , role: IS_AUTHENTICATED_ANONYMOUSLY }
+    +       - { path: "%sylius.security.new_api_shop_regex%/.*", role: ROLE_USER }
```

Comment:
```suggestion
    # config/packages/security.yaml
    security:
        access_control:
            - { path: "%sylius.security.new_api_shop_regex%/.*", role: IS_AUTHENTICATED_ANONYMOUSLY }
            - { path: "%sylius.security.new_api_shop_regex%/orders", method: POST, GET , role: IS_AUTHENTICATED_ANONYMOUSLY }
            - { path: "%sylius.security.new_api_shop_regex%/orders/.*/items", method: POST , role: IS_AUTHENTICATED_ANONYMOUSLY }
            - { path: "%sylius.security.new_api_shop_regex%/.*", role: ROLE_USER }
```

Reply:
something went crazy in here 👯

Reply:
Not really crazy - i just wanted to show like a git diff that first line to be removed and other added. But looks like it is not possible in this docs :/
I will still fix this up with suggestions.